index.html

```diff
@@ -0,0 +1,299 @@
+                                <label for="name">
+                                    <input type="text" id="name" class="form-data"  name="user_name" placeholder="Full name" maxlength="30" required>
+                                </label>
+                            </li>
+                            <li>
+                                <label for="mail">
+                                    <input type="email" id="mail" class="form-data" name="user_email" placeholder="Email address" required>
+                                </label>
+        
+                            </li>
+                            <li>
+                                <label for="msg">
+                                    <textarea name="user_message" class="form-data" id="msg" cols="30" rows="5" maxlength="500" placeholder="Enter text here" required></textarea>
+                                </label>
+                            </li>
+                            <li><small class="error-message" id="errorMessage"></small></li>
+                            <li class="button back-image">
+                                <button type="submit" class="btn">Get in touch</button>
+                            </li>
+        
+                        </ul>
+        
+                    </form>
+        
+                        </div>
+                    </section>
+       
+                </section>
+
+                <section class="contact"> 
+            
+                    <section class="form-section">
+                    
+                        <div class="form-description">
+                        <p class="touch">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+                        </div>
+                    
+                        <div class="contact-form">
+                     <form action="https://formspree.io/f/mjvzwlqy" method="post">
+                       
+                        <ul>
+                            
+                            <li >
+                                <label for="name">
+                                    <input type="text" id="name"  name="user_name" placeholder="Full name" maxlength="30" required>
+                                </label>
+                            </li>
+                            <li>
+                                <label for="mail">
+                                    <input type="email" id="mail" name="user_email" placeholder="Email address" required>
+                                </label>
+        
+                            </li>
+                            <li>
+                                <label for="msg">
+                                    <textarea name="user_message" id="msg" cols="30" rows="5" maxlength="500" placeholder="Enter text here" required></textarea>
+                                </label>
+                            </li>
+                            <li class="button back-image">
```

Comment:
- [ ] Well done, your form works fine, but consider making a few touches to fix the design, see these spaces on the left and right.
- [ ] Your placeholder misses some left spaces, please fix it too
- [ ] Your image after the form bottom right should touch the right side, please take a closer look at the original design to make the necessary changes.

<img width="771" alt="1" src="https://user-images.githubusercontent.com/77843050/213683577-ed7ec53b-9b6b-436d-aac7-bc42de2db7c7.png">